lib/Registrator.js

```diff
@@ -165,10 +166,21 @@ ${this._contact};expires=${this._expires}${this._extraContactParams}`);
               .reduce((a, b) => a.concat(b.parsed), []);
 
             // Get the Contact pointing to us and update the expires value accordingly.
-            const contact = contacts.find((element) => (
-              element.uri.user === this._ua.contact.uri.user
+            // If we have a reg-id and a sip.instance parameter in our Contact header then try
+            // to find a matching Contact using that.
```

Comment:
Cosmetic, this does not apply anymore because reg-id and sip-instance are always there.

` Try to find a matching Contact using sip.instance and reg-id.`

Reply:
Ok, done.